ironic.conf.j2

```diff
@@ -77,12 +80,15 @@ fast_track = {{ env.IRONIC_FAST_TRACK }}
 dhcp_provider = none
 
 [inspector]
-endpoint_override = http://{{ env.IRONIC_URL_HOST }}:5050
+endpoint_override = {{ env.IRONIC_INSPECTOR_BASE_URL }}:5050
```

Comment:
If we're going to call this a URL, I kind of feel like the port number ought to be moved inside it.

Reply:
ok, I will update, thanks